oak_functions/lookup_data_checker/README.md

```diff
@@ -5,5 +5,5 @@ The binary in this directory checks the correctness of the lookup data file.
 Usage:
 
 ```bash
-./scripts/docker_run RUST_LOG=debug cargo run --manifest-path=oak_functions/lookup_data_checker/Cargo.toml -- --file-path=weather_data_file
+./scripts/docker_run "RUST_LOG=debug cargo run --manifest-path=oak_functions/lookup_data_checker/Cargo.toml -- --file-path=weather_data_file"
```

Comment:
@ipetr0v Do you know where the `weather_data_file` lives now?

Reply:
I think it should be this one: https://github.com/project-oak/oak/blob/main/oak_functions/examples/weather_lookup/testdata/lookup_data_weather_sparse_s2

Reply:
Thank you!